### PR TITLE
MM-29712 - "Last updated" time in RHS Incident list view

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Add last updated line in rhs shortlist

- Looks like:
image

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-29712

### CODE REVIEW

File: webapp/src/components/rhs/rhs_list_view.tsx [hunk 1]
```diff
@@ -146,6 +148,8 @@ const RHSListView = () => {
     const currentTeam = useSelector<GlobalState, Team>(getCurrentTeam);
     const currentChannelId = useSelector<GlobalState, string>(getCurrentChannelId);
     const incidentList = useSelector<GlobalState, Incident[]>(myActiveIncidentsList);
+    type GetPostType = (postId: string) => Post;
+    const getPostFromState = useSelector<GlobalState, GetPostType>((state) => (postId) => getPost(state, postId));
```
Comment (cpoile) [inline webapp/src/components/rhs/rhs_list_view.tsx:152]: Hat tip to @agarciamontoro!

**Review by crspeller — APPROVED** [review]

File: webapp/src/components/duration.tsx [hunk 2]
```diff
@@ -5,8 +5,9 @@ import React, {FC, useEffect, useState} from 'react';
 import moment from 'moment';
 
 interface DurationProps {
-    created_at: number;
-    ended_at: number;
+    from: number;
+    to: number;
```
Comment (agarciamontoro) [inline webapp/src/components/duration.tsx:9]: Can we add a comment here stating that `0` means now? It was not super obvious to me in the first place.

**Review by agarciamontoro — APPROVED** [review]
Cool! :tada: I left a non-blocking comment below.

File: webapp/src/components/duration.tsx (hunk 2 above)
Comment (cpoile) [reply webapp/src/components/duration.tsx:9]: Sure, sounds good! Done

**Pull request merged**